.github/workflows/ci-react-compat.yml

```diff
@@ -0,0 +1,30 @@
+name: ci-react-compat
+on:
+  push:
+    branches: [main]
+    paths:
+      - 'packages/react-compat/**'
+  pull_request:
+    branches: ['*']
+    paths:
+      - 'packages/react-compat/**'
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 18.x
```

Comment:
Shall we use `20.x`?

Reply:
Let's use `18.x` in this PR for [the reason](https://github.com/honojs/middleware/pull/1067#pullrequestreview-2719920080).